src/pages/UserPage/UserPage.tsx

```diff
@@ -130,13 +130,25 @@ const UserPage: FC = () => {
       name: 'notify_user',
       label: t('label.user_activation_notification'),
       ariaLabel: t('label.user_activation_notification'),
-      rules: {
-        required: true,
-      },
       className: classes.checkboxInputClass,
     },
   ]
 
+  const onArchiveSubmit: SubmitHandler<FormValues> = useCallback(async () => {
+    const payload: UserStatusType = { institution_user_id }
+    navigate('/settings/users')
```

Comment:
**[blocker]**

We shouldn't navigate away from the screen before we know whether the user archiving was successful or not.